api/routes.py

```diff
@@ -22,7 +22,7 @@ async def upload_file(
     collection_name: MetricCollections,
     file: UploadFile = File(...),
     limit: Optional[int] = SEARCH_RESPONSE_LIMIT,
-):
+) -> list[Any]:
```

Comment:
We know whats the return type, also its provided above in response_model which is used for generating docs (swagger/retool)